engine/client/cl_parse.c

```diff
@@ -3127,6 +3154,7 @@ void CL_ParseLegacyServerMessage( sizebuf_t *msg, qboolean normal_message )
 			break;
 		case svc_voicedata:
 			CL_ParseVoiceData( msg );
```

Comment:
Will legacy servers send `svc_voicedata`?

Highly doubt about it.

Reply:
Yes, I think too, will remove this